web/packages/teleport/src/Integrations/status/AwsRa/AwsRaDashboard.story.tsx

```diff
@@ -0,0 +1,201 @@
+const ctx = createTeleportContext();
+const initialEntries = [
+  cfg.getIntegrationStatusRoute(IntegrationKind.AwsRa, raName),
+];
+const path = cfg.routes.integrationStatus;
+
+export function Dashboard() {
+  return (
+    <ContextProvider ctx={ctx}>
+      <MemoryRouter initialEntries={initialEntries}>
+        <InfoGuidePanelProvider>
+          <Route path={path}>
+            <AwsRaDashboard />
+          </Route>
+        </InfoGuidePanelProvider>
+      </MemoryRouter>
+    </ContextProvider>
+  );
+}
+Dashboard.parameters = {
+  msw: {
+    handlers: [
+      http.get(cfg.getIntegrationsUrl(raName), () => {
+        return HttpResponse.json({
+          name: raName,
+          subKind: 'aws-ra',
+          awsra: {
+            trustAnchorARN: trustAnchorArn,
+            profileSyncConfig: {
+              enabled: true,
+              profileArn:
+                'arn:aws:rolesanywhere:us-east-2:111:profile/31a77874-9f41-4c10-aeae-2aa1140e8ca5',
+              roleArn: 'arn:aws:iam::111:role/ra-int',
+              filters: ['*'],
+            },
+          },
+        });
+      }),
+      http.get(cfg.getIntegrationStatsUrl(raName), () => {
+        return HttpResponse.json({
+          name: raName,
+          subKind: 'aws-ra',
+          awsra: {
+            trustAnchorARN: trustAnchorArn,
+            profileSyncConfig: {
+              enabled: true,
+              profileArn:
+                'arn:aws:rolesanywhere:us-east-2:111:profile/31a77874-9f41-4c10-aeae-2aa1140e8ca5',
+              roleArn: 'arn:aws:iam::111:role/ra-int',
+              filters: ['*'],
+            },
+          },
+          unresolvedUserTasks: 0,
+          awsec2: {},
+          awsrds: {},
+          awseks: {},
+          rolesAnywhereProfileSync: {
+            enabled: true,
+            status: 'SUCCESS',
+            syncedProfiles: 3,
+            syncStartTime: '2025-08-25T17:57:29.5993537Z',
+            syncEndTime: '2025-08-25T17:57:30.538177396Z',
+          },
+        });
+      }),
+    ],
+  },
+};
+
+export function Failed() {
+  return (
+    <ContextProvider ctx={ctx}>
+      <MemoryRouter initialEntries={initialEntries}>
+        <InfoGuidePanelProvider>
+          <Route path={path}>
+            <AwsRaDashboard />
+          </Route>
+        </InfoGuidePanelProvider>
+      </MemoryRouter>
+    </ContextProvider>
+  );
+}
+Failed.parameters = {
+  msw: {
+    handlers: [
+      http.get(cfg.getIntegrationsUrl(raName), () => {
+        return HttpResponse.json(
+          {
+            error: { message: 'Generic Bad Request' },
+          },
+          { status: 400 }
+        );
+      }),
+      http.get(cfg.getIntegrationStatsUrl(raName), () => {
+        return HttpResponse.json(
+          {
+            error: { message: 'Generic Bad Request' },
+          },
+          { status: 400 }
+        );
+      }),
+    ],
+  },
+};
+
+export function NoData() {
```

Comment:
this one gives an error (and i tried refreshing several times to make sure it's not storybook quirking)